ngraph/test/frontend/paddlepaddle/op/concat.cpp

```diff
@@ -0,0 +1,79 @@
+                                                              {1.0, 1.0, 1.0, 1.0}}}}}.get_vector());
+
+    res.inputs.emplace_back(test::NDArray<float, 4>{{{{{0.0, 0.0, 0.0, 0.0},
+                                                              {0.0, 0.0, 0.0, 0.0},
+                                                              {0.0, 0.0, 0.0, 0.0},
+                                                              {0.0, 0.0, 0.0, 0.0}}}}}.get_vector());
+
+
+
+    res.expected_outputs.emplace_back(test::NDArray<float, 4>{{{{1.0, 1.0, 1.0, 1.0},
+                                                                       {1.0, 1.0, 1.0, 1.0},
+                                                                       {1.0, 1.0, 1.0, 1.0},
+                                                                       {1.0, 1.0, 1.0, 1.0},
+                                                                       {0.0, 0.0, 0.0, 0.0},
+                                                                       {0.0, 0.0, 0.0, 0.0},
+                                                                       {0.0, 0.0, 0.0, 0.0},
+                                                                       {0.0, 0.0, 0.0, 0.0}}}}.get_vector());
+    return res;
+}
+
+static concatTestParam paddle_concat_minus_axis() {
+    concatTestParam res;
+    res.m_frontEndName = PDPD;
+    res.m_modelsPath =   PATH_TO_MODELS;
+    res.m_modelName =    "concat_minus_axis";
+
+    res.inputs.emplace_back(test::NDArray<float, 4>{{{{{1.0, 1.0, 1.0, 1.0},
+                                                              {1.0, 1.0, 1.0, 1.0},
+                                                              {1.0, 1.0, 1.0, 1.0},
+                                                              {1.0, 1.0, 1.0, 1.0}}}}}.get_vector());
+
+    res.inputs.emplace_back(test::NDArray<float, 4>{{{{{0.0, 0.0, 0.0, 0.0},
+                                                              {0.0, 0.0, 0.0, 0.0},
+                                                              {0.0, 0.0, 0.0, 0.0},
+                                                              {0.0, 0.0, 0.0, 0.0}}}}}.get_vector());
+
+
+
+    res.expected_outputs.emplace_back(test::NDArray<float, 4>{{{{{1.00, 1.00, 1.00, 1.00, 0.00, 0.00, 0.00, 0.00 },
+                                                                       {1.00, 1.00, 1.00, 1.00, 0.00, 0.00, 0.00, 0.00 },
+                                                                       {1.00, 1.00, 1.00, 1.00, 0.00, 0.00, 0.00, 0.00 },
+                                                                       {1.00, 1.00, 1.00, 1.00, 0.00, 0.00, 0.00, 0.00 }}}}}.get_vector());
+    return res;
+}
+
+TEST_P(concatTest, test_concat) {
+    ASSERT_NO_THROW(validateOp());
+}
+
+INSTANTIATE_TEST_CASE_P(FrontendOpTest, concatTest,
+                        ::testing::Values(
+                                concat_axis(),
+                                concat_minus_axis()
```

Comment:
compile error. Should it be 'paddle_concat_minus_axis'?

Reply:
I remove this useless unit test